portfolio/src/main/webapp/script.js

```diff
@@ -67,9 +68,18 @@ function createCommentElement(text) {
   return commentElement;
 }
 
-/** Clears out old comments and inserts the selected number of comments. */
-function changeCommentsNum(){
-  document.getElementById("history").innerHTML = "";
+/** Changes number of comments displayed. */
+function changeCommentsNum() {
+  clearComments();
   getComments();
 }
 
+/** Tells the server to delete all comments. */
```

Comment:
The 'tells' is a bit redundant here, I'd always document what the function does rather than "it's story", for example "Handles comments deletion".